SingularityUI/app/assets/_index.mustache

```diff
@@ -27,6 +27,9 @@
             hideNewRequestButton: {{{hideNewRequestButton}}},
             defaultCpus: {{{defaultCpus}}},
             defaultMemory: {{{defaultMemory}}},
+            defaultHealthcheckIntervalSeconds: {{{defaultHealthcheckIntervalSeconds}}},
+            defaultHealthcheckTimeoutSeconds: {{{defaultHealthcheckTimeoutSeconds}}},
+            defaultDeployHealthTimeoutSeconds: {{{defaultDeployHealthTimeoutSeconds}}},
```

Comment:
Please add defaults for these in [`config.coffee`](https://github.com/HubSpot/Singularity/blob/master/SingularityUI/config.coffee). Without them you're breaking the Brunch-served index file.